### PR TITLE
Fix/use correct type names

### PR DESCRIPTION
**Pull request by Redmega**
Adds a bunch of tests and fixes some pluralization issues

### CODE REVIEW

**Comment by Redmega** [comment]
When this is merged I will publish to the @next tag so we can start testing on example sites w/o ruining anyones days.

Publishing this will mark the release of `v1.0.0`

File: src/__tests__/faulty-keywords.test.js [hunk 1]
```diff
@@ -1,27 +1,29 @@
...
+    expect(
+      checkForFaultyFields(faultyResponse)
+    ).toBeTruthy();
+  });
 
-it(`is falsy if the faulty keyword is aliased`, () => {
-  expect(
-    checkForFaultyFields(swellResponse)
-  ).toBeFalsy();
+  it(`ignores faulty keywords if they are aliased`, () => {
```
Comment (Redmega) [inline src/__tests__/faulty-keywords.test.js]: Would even say this is not even the test case. It's more like... `detects when there is no faulty keywords` or something like that. Nothing in the code is actually checking if its a keyword which was aliased...

**Comment by rdela** [comment]
If you can build a site with it I am happy to have you push to npm and try from there, local builds are error below but could be something I am doing.

https://app.netlify.com/sites/kokodokai/deploys/5aa1c954295b550a0ac6aa8c
https://github.com/rdela/gatsby-graphcms-rd/tree/canary

**Comment by rdela** [comment]
vs a successful deploy

https://app.netlify.com/sites/kokodokai/deploys/5aa1976eefbe5d48095be0b4

**Comment by rdela** [comment]
so I would say it fails on `onCreateNode: Artists`

**Comment by rdela** [comment]
any idea why?

**Comment by rdela** [comment]
Do I have the wrong (non-)plural somewhere now?

File: src/__tests__/util.test.js [hunk 2]
```diff
@@ -47,7 +47,7 @@ describe('Util function tests', () => {
       expect(constructTypeQuery(type).replace(/\s/g, '')).toBe(query);
     });
 
-    // TODO: The rest of the assembleQueries tests.
+    // Next: The rest of the assembleQueries tests.
```
Comment (Redmega) [inline src/__tests__/util.test.js]: NooO!!!!

I wanted to use comment tags that have semantic highlighting 😭 

(I don't personally mind the warnings. They're a pretty shade of yellow, don't you think?)

**Comment by Redmega** [comment]
Alright I'll test out on my local in an hour or so

**Comment by Redmega** [comment]
So all the issues were caused by pluralization assumptions that were _wrong_.

Apparently, for a given type `Artist`, its singular within gatsby graphql queries is `artist` and its plural is `allArtist`. This is read as "All of type Artist" which makes some kind of sense I guess...

File: src/__tests__/util.test.js (hunk 2 above)
Comment (rdela) [reply src/__tests__/util.test.js]: lovely shade fine with me!!!

**Pull request merged**